### PR TITLE
Fix print version of CertTemplate properties

### PR DESCRIPTION
**Pull request by JonasBK**
## Description

Fix the strings representation shown in the entity panel of cert template properties

## Motivation and Context

Because we like it pretty.

## Screenshots (if appropriate):

Before:
image

After:
image

## Types of changes

-   [x] Chore (a change that does not modify the application functionality)
-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

-   [ ] Documentation updates are needed, and have been made accordingly.
-   [ ] I have added and/or updated tests to cover my changes.
-   [x] All new and existing tests passed.
-   [ ] My changes include a database migration.

### CODE REVIEW

File: packages/cue/bh/ad/ad.cue [hunk 1]
```diff
@@ -394,10 +394,66 @@ AuthenticationEnabled: types.#StringEnum & {
 EnrolleeSuppliesSubject: types.#StringEnum & {
 	symbol: "EnrolleeSuppliesSubject"
 	schema: "ad"
-	name: "Enrollee Suppliess Subject"
```
Comment (urangel) [inline packages/cue/bh/ad/ad.cue:397]: Nice typo find 🙌

File: packages/cue/bh/ad/ad.cue [hunk 2]
```diff
@@ -394,10 +394,66 @@ AuthenticationEnabled: types.#StringEnum & {
...
+	name: "Certificate Name Flags"
+	representation: "certificatenameflag"
+}
+
+EffectiveEKUs: types.#StringEnum & {
+	symbol: "EffectiveEKUs"
+	schema: "ad"
+	name: "Effective EKUs"
+	representation: "effectiveekus"
+}
```
[the remark this answers is not in the document]
Comment (JonasBK) [reply packages/cue/bh/ad/ad.cue:420]: Well spotted! The effective EKUs can either be the content of the pKIExtendedKeyUsage attribute or msPKI-Certificate-Application-Policy:
image
I created EffectiveEKUs so the user doesn't have to learn that logic

File: packages/cue/bh/ad/ad.cue [hunk 3]
```diff
@@ -394,10 +394,66 @@ AuthenticationEnabled: types.#StringEnum & {
...
-	name: "Enrollee Suppliess Subject"
+	name: "Enrollee Supplies Subject"
 	representation: "enrolleesuppliessubject"
 }
 
+CertificateApplicationPolicy: types.#StringEnum & {
+	symbol: "CertificateApplicationPolicy"
+	schema: "ad"
+	name: "Certificate Application Policies"
+	representation: "certificateapplicationpolicy"
```
[the remark this answers is not in the document]
Comment (JonasBK) [reply packages/cue/bh/ad/ad.cue:405]: Yup. CertificateApplicationPolicy matches the AD attribute msPKI-Certificate-Application-Policy. If you view it in the GUI in AD, it is plural. So I thought we could do the same.

File: packages/cue/bh/ad/ad.cue (hunk 2 above)
[the remark this answers is not in the document]
Comment (urangel) [reply packages/cue/bh/ad/ad.cue:420]: Ah so they can be different. Okay sounds good, thanks for the response.

File: packages/cue/bh/ad/ad.cue (hunk 3 above)
[the remark this answers is not in the document]
Comment (urangel) [reply packages/cue/bh/ad/ad.cue:405]: Huh, okay that makes sense. Thanks for the info!

**Review by urangel — APPROVED** [review]
lgtm

**Pull request merged**